ocaml/typing/typeclass.ml

```diff
@@ -1281,8 +1281,7 @@ and class_expr_aux cl_num val_env met_env virt self_scope scl =
               Arg (
                 if not optional || Btype.is_optional l' then
                   let arg = type_argument val_env sarg ty ty0 in
-                  arg, Ctype.type_sort_exn ~why:Function_argument
-                         arg.exp_env arg.exp_type
+                  arg, Sort.value
```

Comment:
`Sort.for_class_parameter`. I should have been more specific in my earlier comment. It's conceivable we'll relax this someday.

Reply:
I updated all the sorts in transl to use the new aliases, but not in the typechecker.  My instinct is that would be better done in a comprehensive pass that just removes `Sort.value`.

Reply:
Right. I remember that now. OK.